README.md

```diff
@@ -291,6 +297,24 @@ let arr_instance = jvm.create_java_array("java.lang.String", &[s1, s2, s3])?;
 let list_instance = jvm.invoke_static("java.util.Arrays", "asList", &[InvocationArg::from(arr_instance)])?;
 ```
 
+When creating an array of primitives, each item needs to be converted first.
```

Comment:
In order to be more clear, can we change this to:

When creating an array of primitives, each `InvocationArg` item needs to be converted to one containing primitive value.

Reply:
Done.